client/js/libs/handlebars/localetime.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+const moment = require("moment");
+
 module.exports = function(time) {
-	return new Date(time).toLocaleString();
+	return moment(time).format("MMMM DD, YYYY – HH:mm:ss");
```

Comment:
Similar comment here and above. While the previous change (`L` to `MMMM DD, YYYY`) forces an American standard, this one will change from `4:30:00 AM` to `16:30:00`, actually parting from the American standard. So even if we forget about internationalization and go for an opinionated change, there is a loss of consistency in the standard used.
(Though currently tooltip show `4:30:00 AM` but message time shows `16:30`... Ah, if only we had proper localization :D)